src/ol/worker/version.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * @module ol/worker/version
+ * A worker that responds to messages by posting a message with the version identifer.
+ */
+import {VERSION} from '../util';
+
+onmessage = event => {
+  console.log('version worker received message:', event.data); // eslint-disable-line
+  // @ts-ignore
```

Comment:
This `@ts-ignore` can be removed if `webworker` is added in `tsconfig.json`, see https://github.com/fredj/openlayers/commit/bb022050edeef790c9cb505cce255702295fde53

Reply:
Feel free to add this commit (now or later) if this fixes the warning.  I was running into an issue with a d.ts file that had an outdated signature.  Can’t find the github issue now, but it looked like it was fixed upstream.

Reply:
I've pushed the commit, thanks